inference-engine/tests/functional/plugin/cpu/subgraph_tests/src/add_conver_to_reorder.cpp

```diff
@@ -70,7 +70,7 @@ TEST_F(AddConvertToReorderTest, smoke_TestAddConvert_CPU) {
     BuildGraph(ngraph::element::bf16);
     Run();
     CheckNodeOfTypeCount(executableNetwork, "Convert", 1);
-    CheckNodeOfTypeCount(executableNetwork, "Reorder", 0);
+    CheckNodeOfTypeCount(executableNetwork, "Reorder", 1);
```

Comment:
Why the number was increased?

Reply:
axis node presented as real node now
reorder inserted for it